sdk/tracing/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpanProcessor.java

```diff
@@ -142,169 +132,148 @@ public void forceFlush() {
     worker.forceFlush();
   }
 
+  // TODO remove this when this.forceFlush returns CompletableResultCode
+  @VisibleForTesting
+  CompletableResultCode flush() {
+    return worker.forceFlush();
+  }
+
   // Worker is a thread that batches multiple spans and calls the registered SpanExporter to export
   // the data.
-  //
-  // The list of batched data is protected by an explicit monitor object which ensures full
-  // concurrency.
   private static final class Worker implements Runnable {
 
     static {
       Meter meter = OpenTelemetry.getMeter("io.opentelemetry.sdk.trace");
-      LongCounter droppedSpansCounter =
+      LongCounter processedSpansCounter =
           meter
-              .longCounterBuilder("droppedSpans")
+              .longCounterBuilder("processedSpans")
               .setUnit("1")
               .setDescription(
                   "The number of spans dropped by the BatchSpanProcessor due to high throughput.")
```

Comment:
this description isn't quite right any more. Can be fixed in a follow-on PR, to get you unblocked.